metalearners/drlearner.py

```diff
@@ -398,3 +405,44 @@ def _pseudo_outcome(
             )
 
         return pseudo_outcome
+
+    @classmethod
+    def necessary_onnx_models(cls) -> set[str]:
+        return {TREATMENT_MODEL}
+
+    @copydoc(MetaLearner.build_onnx, sep="")
+    def build_onnx(self, models: Mapping[str, Sequence], output_name: str = "tau"):
+        """In the DRLearner case, the necessary models are:
+
+        * ``"treatment_model"``
+        """
+        warning_experimental_feature("build_onnx")
+        check_onnx_installed()
+        check_spox_installed()
```

Comment:
Spox depends on `onnx` so only the latter check should be necessary.

Reply:
[de5d495](https://github.com/Quantco/metalearners/pull/39/commits/de5d495b4fedc2a32400a74a138b08c8de69bbce)